apps/sms/js/thread_list_ui.js

```diff
@@ -140,14 +140,8 @@ var ThreadListUI = {
     this.cancelEdit();
   },
 
-  getAllInputs: function thlui_getAllInputs() {
-    if (this.container) {
-      return Array.prototype.slice.call(
-        this.container.querySelectorAll('input[type=checkbox]')
-      );
-    } else {
-      return [];
-    }
+  getIdIterator: function thlui_getIdIterator() {
+    return Threads.getAll().keys();
```

Comment:
this is the only place we use getAll(); so maybe it's better to have a Threads.keys() instead?